src/redux/slices/traineeProfileSlice.ts

```diff
@@ -52,7 +53,17 @@ const traineeProfileSlice = createSlice({
       })
       .addCase(fetchTraineeProfileData.fulfilled, (state, action) => {
         state.status = "succeeded";
-        state.traineeProfileData = action.payload;
+        state.traineeProfileData.traineeTisId = action.payload.traineeTisId;
+        state.traineeProfileData.personalDetails =
+          action.payload.personalDetails;
+        state.traineeProfileData.programmeMemberships =
+          action.payload.programmeMemberships;
```

Comment:
Do we not need to sort programme memberships too?

Reply:
@Judge40 Good shout. Did leave as-is because couldn't find any sort on the Programmes data in the current code.  Either way, would be good to get some more use out of the genericSort. (change could even be as a v small follow-up PR! 😁)